htdocs/core/triggers/interface_20_all_Logevents.class.php

```diff
@@ -158,6 +158,13 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 			// Initialisation donnees (date,duree,texte,desc)
 			$text = $langs->transnoentities("GroupDeleted", $object->name);
 			$desc = $langs->transnoentities("GroupDeleted", $object->name);
+		} elseif ($action == 'USER_IBANMODIFY') {
```

Comment:
We try to reduce the nb of trigger events. Must match the CRUD. So Create , Modify, Delete.
Here changing the ban for a user is doing an update of the UserBankAccount. So you must use the trigger into the update event of UserBankAccount so USERBANKACCOUNT_MODIFY if you need to generate a particular track in agenda.